### PR TITLE
Nerfs .45 bullets and buffs 9mm bullets.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
- balance: ".45 bullets now have no armor penetration."
  - balance: "9mm bullets now do less base damage than .45 bullets, but have a good amount of AP."

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -121,11 +121,12 @@
...
-	armor_penetration = 10
+	damage = 20
+	armor_penetration = 15
 
 /obj/item/projectile/bullet/pistol/medium
 	damage = 30
+	armor_penetration = 0
 
 /obj/item/projectile/bullet/pistol/medium/ap
 	armor_penetration = 35
```
Comment (SleepyGemmy) [inline code/modules/projectiles/projectile/bullets.dm:132]: consider nerfing this as well.

**Review by SleepyGemmy — APPROVED** [review]

File: code/modules/projectiles/projectile/bullets.dm (hunk 1 above)
Comment (NonQueueingMatt) [reply code/modules/projectiles/projectile/bullets.dm:132]: Basically unused

Comment (SleepyGemmy) [reply code/modules/projectiles/projectile/bullets.dm:132]: it's used for the mech SMG and will probably almost instantly kill anyone the few times its used.

Comment (NonQueueingMatt) [reply code/modules/projectiles/projectile/bullets.dm:132]: Good catch

**Pull request merged**